components/contract-components/contract-deploy-form/custom-contract.tsx

```diff
@@ -91,8 +91,8 @@ const CustomContractForm: React.FC<CustomContractFormProps> = ({
     defaultValues: {
       addToDashboard: true,
       deployParams: deployParams.reduce((acc, param) => {
-        acc[param.name] = replaceTemplateValues(
-          fullPublishMetadata.data?.constructorParams?.[param.name]
+        acc[param.name || ""] = replaceTemplateValues(
```

Comment:
@joaquim-verges I thought we fixed the param.name always being there?

Reply:
needs to be schema.parsed i think

Reply:
doesn't work either

Reply:
fixed via various PRs to the sdk